src/scroll_list/AwesomeMapFactory.js

```diff
@@ -188,13 +192,17 @@ define(function(require) {
                 if (options.scaleLimits) {
                     map.addInterceptor(new ScaleInterceptor(options.scaleLimits));
                 }
-                map.addInterceptor(new BoundaryInterceptor({
+                var boundaryInterceptor = new BoundaryInterceptor({
                     centerContentX: hAlignCenter,
                     centerContentY: vAlignAuto,
                     mode: { x: 'stop', y: 'slow' },
                     pinToLeft: hAlignLeft,
                     pinToTop: vAlignTop
-                }));
+                });
+                boundaryInterceptor.onScrollPastBoundary(function(source, args) {
+                    scrollList._shouldPropagateBoundaryEvent(source,args);
+                });
+                map.addInterceptor(boundaryInterceptor);
```

Comment:
Yeah :(  Similar but slightly different enough to make it annoying/difficult to fix